ha_install.sh

```diff
@@ -202,14 +217,16 @@ mv \
   __init__.py \
   alarm_control_panel \
   alert \
-  alexa \
+  alexa \       
+  asuswrt \
```

Comment:
Does all these components required for xiaomi gateway3? 
I'd like to avoid the final installation to be bloated.
Though, telegram bot and  xiaomi gw3 i find reasonable.